tools/engine_tool/lib/src/build_utils.dart

```diff
@@ -65,6 +65,41 @@ void debugCheckBuilds(List<Build> builds) {
   }
 }
 
+/// Transform the name of a build into the name presented and accepted by the
+/// CLI
+///
+/// If a name starts with '$OS/', it is a local development build, and the
+/// mangled name has the '$OS/' part stripped off, where $OS is the value of
+/// `platform.operatingSystem` in the passed-in `environment`.
+///
+/// If the name does not start with '$OS/', then it must start with 'ci/' or
+/// 'ci\' in which case the name is returned unchanged.
```

Comment:
some example inputs/outputs would help this comment

Reply:
Done.